scipy/stats/_continuous_distns.py

```diff
@@ -7166,6 +7166,92 @@ def _sf(self, x, chi):
 argus = argus_gen(name='argus', longname="An Argus Function", a=0.0, b=1.0)
 
 
+class irwinhall_gen(rv_continuous):
+    r"""
+    Irwin Hall distribution
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The `irwinhall` cumulative distribution function is:
+
+    .. math::
+
+        F(x) = \frac{1}{n!}\sum_{k=0}^{\lfloor x\rfloor}
```

Comment:
Normally the docstring states the pdf at the beginning, though not very informative in my view. You should definitely say that the distribution  the sum of n uniform and independent rvs

Reply:
I switched the docstring to state the definition of `f(x)` the pdf to be consistent with other distributions. I also added 1 line to state the distribution is sum of `n` independent uniform(0,1) r.v.s.